draft-irtf-cfrg-hpke.md

```diff
@@ -1031,6 +1031,19 @@ security models and assumptions, and do not consider attackers capable of quantu
 computation. A full proof of post-quantum security would need to take this
 difference into account, in addition to simply using a post-quantum KEM.
 
+## Key Compromise Impersonation
+
+Both the Auth and AuthPSK modes of DHKEM are susceptible to Key Compromise
+Impersonation (KCI) attacks {{?BJM97=DOI.10.1007/BFb0024447}}. Specifically,
+if an adversary compromises `skR` for some receiver, then the adversary can
+forge new messages from any `pkS` to that receiver. These ciphertexts will
+decrypt properly even though they were not produced by an entity holding `skS`.
```

Comment:
Do you want to note that predecessors such as libnacl Box also have this issue?

Reply:
Yeah, let's do it. @blipp, can you do this in your PR?